src/js/tracing/reactnativeprofiler.tsx

```diff
@@ -47,12 +44,7 @@ export class ReactNativeProfiler extends Profiler {
     }
 
     client.addIntegration && client.addIntegration(createIntegration(this.name));
-
-    const endTimestamp = this._mountSpan && typeof spanToJSON(this._mountSpan).timestamp
-    const tracingIntegration = client.getIntegrationByName && client.getIntegrationByName<ReactNativeTracing>('ReactNativeTracing');
-    tracingIntegration
-      && typeof endTimestamp === 'number'
-      // The first root component mount is the app start finish.
-      && tracingIntegration.onAppStartFinish(endTimestamp);
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    captureAppStart();
```

Comment:
Q:Shouldn't we wait for it?

Reply:
We only trigger the capture of app start here, recording the timestamp and we don't await it as the fetch from native layer can happen async later